data/remnant/remnant missions.txt

```diff
@@ -872,13 +876,89 @@ mission "Remnant: Technology Available"
-			`	"When your name came up in the study," he says, "I asked around and discovered that you have helped us in other ways as well. The other prefects have reached a consensus that you should be considered a valuable friend of the Remnant, and shall henceforth be given permission to purchase our ships and technology."`
+			`	"When your name came up in the study," he says, "I asked around and discovered that you have helped us in other ways as well. The other prefects have reached a consensus that you should be considered a valuable friend of the Remnant, and shall henceforth be given permission to purchase our ships and technology." He hands you a small cube that looks like it is made of interlocking blocks of material similar to their ships.`
+			scene "images/outfit/remnant license"
 			choice
 				`	"I'm honored. Thank you."`
-					decline
+					goto onwards
 				`	"Is there anything else I can do to assist your people?"`
-			`	"Not that I am aware of right now," he says, "but we will call on you if a need ever arises."`
-				decline
+			`	"I do not have anything at the moment," he says, "but I am sure other Remnant will have tasks you can help with, now that you have demonstrated your trustworthiness. We have been keeping track of you, and we are impressed with your curiosity, perseverance, and ability to survive where most would be destroyed. All three are valued characteristics among the Remnant, and thus many will welcome your assistance on their projects."`
+			label onwards
+			`	As he talks, he gestures for you to walk with him towards a nearby hall. Inside, you discover that it is one of the many cafeterias that seem to be common in Remnant starports. He picks up two trays from the window and leads you to an empty table. "While you are welcome to become one of us, we suspect that you are a citizen of the galaxy - wandering hither and yon building coalitions and enmities with species and groups across the breadth of accessible space, unfettered by conventional allegiances. As such, we hope that you will continue to explore and stop by to let us know what you have found. We ask only that you keep the secret with which you are trusted and bring us word of anything that might influence humanity's long-term survival."`
+			choice
+				`	"What secret do you want me to keep?`
+					goto secret
+				`	"I would be honored to be one of the Remnant."`
+					goto beremnant
+				`	"While I have enjoyed my time among you, I can never stay in one place."`
+					goto galacticcitizen
+				`	"What constitutes 'things that influence humanity's survival'?"`
+			label humanitiessurvival
```

Comment:
I'd do "humanitiessurvival" -> "survival", "beremnant" -> "remnant", and "galacticcitizen" -> "traveler" for the sake of readability.

Reply:
I prefer doing two word labels to help ensure that they are unique. Makes it easier to search for things and avoid duplicates, etc.

Reply:
I don't.

Reply:
I'll leave it as is, then. I prefer to keep it easier to verify manually just in case.

Reply:
Since readability is my concern, you could change these to `"be remnant"` and so on, but I think that one word will do just fine. (I'd really only suggest multi-word labels for massive conversations where having lengthy labels makes navigation easier, e.g. #4905. With a small conversation like this though, it isn't necessary.)

Reply:
OK, I changed two of them in the interests of readability. I'd prefer to leave `beremnant` alone, though. I don't think that one is a readability problem. And there are far too many instances of remnant in the file.

Replaced humanitiessurvival with survival
Replaced galacticcitizen with traveler

Reply:
"beremnant" -> "join" or "honored"?

Reply:
OK. "beremnant" replaced with "honored"